src/pages/philco.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+  <title>Philco test page</title>
+
+  <link href="../assets/fonts/EuclidTriangle/stylesheet.css" rel="stylesheet" type="text/css" />
+
+  <script type="module" src="../build/biggive.esm.js"></script>
+  <script nomodule src="../build/biggive.js"></script>
+
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+  </style>
+</head>
+<body style="background-color: #F1F1F1;">
+
+<philco-main-menu
+  philco-url-prefix="http://localhost:30003"
+></philco-main-menu>
+
+
+<!--
+todo - work out how to express that the first word of "Transform Capitalism below should be a different colour"
+and the part of the teaser up to the word "philanthropy," should be bold.
+-->
```

Comment:
this coloured words requirement seems quite tricky if we use the existing BG components that take plain text props for title and teaser - would be good to discuss options. Maybe we need to use slots to allow HTML elements with coloured words to be passed in.